setup.cfg

```diff
@@ -115,6 +115,7 @@ ignore_imports =
     # cms.djangoapps.export_course_metadata.tasks
     #  -> openedx.core.djangoapps.schedules.content_highlights
     #  -> lms.djangoapps.courseware.block_render & lms.djangoapps.courseware.model_data
+    openedx.core.djangoapps.content_libraries.signal_handlers -> lms.djangoapps.grades.api
     openedx.core.djangoapps.schedules.content_highlights -> lms.djangoapps.courseware.*
```

Comment:
each ignore line has a comment above it explaining the chain of imports that caused the ignore to be necessary. could you update your new line to follow that?